examples/pipeline_config.yaml

```diff
@@ -13,7 +13,7 @@ exclusion_files:
 stages:
   - function: delete_tables
     run: False
-    prefix: USER_
+    prefix: stef_
```

Comment:
This looks like your personal config. Please could you remove this part and any genuine file paths